apps/eth_blockchain/lib/eth_blockchain/childchain.ex

```diff
@@ -37,6 +37,9 @@ defmodule EthBlockchain.Childchain do
         opts \\ []
       ) do
     with :ok <- check_childchain(childchain_identifier),
```

Comment:
what's the issue in getting rid of this?

Reply:
All of this things are related to plasma deposits which are revamped to support ALD format in this PR: https://github.com/omgnetwork/ewallet/pull/1217/files